esm-collection/json-schema/schema.json

```
@@ -1,14 +1,15 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "schema.json#",
   "title": "ESM Extension Specification",
   "description": "This object represents Collections in a Earth System Model Catalog.",
   "allOf": [
     {
-      "$ref": "https://raw.githubusercontent.com/radiantearth/stac-spec/v1.0.0-beta.1/collection-spec/json-schema/collection.json"
+      "$ref": "#/definitions/catalog"
     },
     {
-      "$ref": "https://raw.githubusercontent.com/radiantearth/stac-spec/v1.0.0-beta.1/extensions/collection-assets/json-schema/schema.json"
+      "$ref": "#/definitions/collection"
```

We started hosting them correctly:

```suggestion
      "$ref": "https://schemas.stacspec.org/v1.0.0-beta.2/collection-spec/json-schema/collection.json"
```